src/components/DeeplinkWrapper/index.website.js

```diff
@@ -12,9 +12,13 @@ const propTypes = {
     children: PropTypes.node.isRequired,
 };
 
-class DeeplinkWrapper extends PureComponent {
```

Comment:
We can use `memo` equivalent to `PureComponent`

Reply:
we don't need memo here

Reply:
I thought no need at first. But this actually includes entire app views as children.
So still you persist no performance optimization here?